Earthfile

```diff
@@ -435,7 +436,7 @@ base-image:
 
     # Set /boot/vmlinuz pointing to our kernel so kairos-agent can use it
     # https://github.com/kairos-io/kairos-agent/blob/0288fb111bc568a1bfca59cb09f39302220475b6/pkg/elemental/elemental.go#L548   q
-    IF [ "$FLAVOR" = "fedora" ] || [ "$FLAVOR" = "rockylinux" ] || [ "$FLAVOR" = "almalinux" ]
+    IF [ "$FLAVOR" = "fedora" ] || [ "$FAMILY" = "rhel" ]
```

Comment:
I would just do a `RUN rm -rf /boot/initramfs-* || true` so it always runs instead of having switches here so we just try to remove it always and not fail.